README.md

```diff
@@ -10,6 +10,7 @@
 # Quick Links
 
 - [About](#about)
+- [Features](#features)
```

Comment:
Ohh I didn't realize these were actually links to inside locations within the readme.. Fancy

Reply:
Oh dang... that IS fancy